src/components/NoteDisplay/index.js

```diff
@@ -0,0 +1,125 @@
+import React, { useContext, useState } from "react";
+import Head from "./NoteDIsplayHead";
+import styled from "styled-components";
+import { AppContext } from "../../contexts/AppContext";
+import { ThemeContext } from "../../contexts/ThemeContext";
+import { useLocation } from "react-router-dom";
+import globalTypes from "../../GlobalTypes";
+import TagContainer from "./TagContainer";
+import Taglist from "./Taglist";
+import TagInput from "./TagInput";
+import Tag from "./Tag";
+import DeleteMode from "./DeleteMode";
+import NonDeleteMode from "./NonDeleteMode";
+
+const Display = styled.div`
+    width: 100%;
+    display: ${(props) => (props.noteId ? "grid" : "none")};
+    grid-template-rows: 12vh 1fr;
+    transform: translateX(${(props) => (props.navbarOpen ? "20vw" : "0")});
+    filter: ${(props) => (props.navbarOpen ? "blur(3px)" : "none")};
+
+    @media only screen and (max-width: 800px) {
+        transform: translateX(0);
+    }
+`;
+
+const DisplayInput = styled.textarea`
+    border: none;
+    padding: 1rem 3rem;
+    font-family: inherit;
+    font-size: inherit;
+    color: inherit;
+    height: 100%;
+    font-size: 1.2rem;
+    background: transparent;
+    width: 100%;
+`;
+
+export default function () {
```

Comment:
Can you add a name to the function as to avoid making it an anonymous one? This will help in debugging